addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -118,9 +126,15 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     break;
                 case REBOOT:
                     setReboot(command);
+                    break;
+                default:
+                    logger.debug("Thing {}: unexpected command {} from channel {}", getThing().getUID(), command,
+                            channelUID.getId());
+                    break;
             }
         } catch (FreeboxException e) {
-            logger.error("{}", e.getMessage());
+            logger.warn("Thing {}: error while handling command {} from channel {}", getThing().getUID(), command,
```

Comment:
Is this situation a bug? Or can we get here in "normal" operation (e.g. on network issues)? If so, setting the Thing status would be the better thing to do.